From e78ee48e5ebe9ce9de64589bd0a928459f3e93a0 Mon Sep 17 00:00:00 2001
From: Demon000 <demonsingur@gmail.com>
Date: Thu, 11 Jan 2018 11:15:25 +0200
Subject: [PATCH 11/20] legacy: label old msm_irqbalance prop

Removed in 512f5a2cc815ef50109091daf98cb6bfa88a2c40,
used by old blobs.

Change-Id: I5794f480f4c1be94a0e762de6889d95499f5ed17
---
 legacy-common/property_contexts | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 legacy-common/property_contexts

diff --git a/legacy-common/property_contexts b/legacy-common/property_contexts
new file mode 100644
index 0000000..c755d81
--- /dev/null
+++ b/legacy-common/property_contexts
@@ -0,0 +1 @@
+ctl.msm_irqbalance         u:object_r:msm_irqbalance_prop:s0
-- 
2.7.4

